index.bs

```diff
@@ -10505,12 +10505,13 @@ The [=remote end steps=] given |session| and |command parameters| are:
 1. If |element| doesn't implement {{Element}}, return [=error=] with [=error code=]
    [=no such element=].
 
-1. If |element| doesn't implement {{HTMLInputElement}},
-   |element|["<code>type</code>"] is not "file", or
-   |element|["<code>disabled</code>"] is true, return [=error=] with [=error
-   code=] [=unable to set file input=].
+1. If |element| doesn't implement {{HTMLInputElement}}, |element|'s
+   <{input/type}> attribute is not "<code>file</code>", or |element|'s <{input/disabled}>
```

Comment:
For the type attribute, we should maybe use https://html.spec.whatwg.org/#file-upload-state-(type=file) i.e. `|element is not in the [=file upload state=]`